content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -183,7 +181,7 @@ Specifying a timezone that way is **not officially supported** (and never has be
 
 If you try to set a schedule that includes `TZ` or `CRON_TZ` timezone specification,
 Kubernetes reports a [warning](/blog/2020/09/03/warnings/) to the client.
-Future versions of Kubernetes might not implement that unofficial timezone mechanism at all.
+Future versions of Kubernetes will prevent setting the unofficial timezone mechanism entirely.
```

Comment:
@sftim in preparation to land https://github.com/kubernetes/kubernetes/pull/116252 I've changed this wording to be stronger.